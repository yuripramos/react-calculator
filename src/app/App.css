@font-face {
  font-family: 'OpenSans Thin';
  src:  url("../assets/fonts/OpenSans/OpenSans-Light.woff2") format('woff2'),
        url('../assets/fonts/OpenSans/OpenSans-Light.woff') format('woff');
}
@font-face {
  font-family: 'OpenSans Regular';
  src:  url("../assets/fonts/OpenSans/OpenSans-Regular.woff2") format('woff2'),
        url('../assets/fonts/OpenSans/OpenSans-Regular.woff') format('woff');
}
@font-face {
  font-family: 'RobotoSlab Regular';
  src:  url("../assets/fonts/Roboto/RobotoSlab-Regular.woff2") format('woff2'),
        url('../assets/fonts/Roboto/RobotoSlab-Regular.woff') format('woff');
}
@font-face {
  font-family: 'RobotoSlab Thin';
  src:  url("../assets/fonts/Roboto/RobotoSlab-Thin.woff2") format('woff2'),
        url('../assets/fonts/Roboto/RobotoSlab-Thin.woff') format('woff');
}
@font-face {
  font-family: 'Helveticaneue Thin';
  src:  url("../assets/fonts/Helveticaneue/helveticaneue.woff2") format('woff2'),
        url('../assets/fonts/Helveticaneue/helveticaneue.woff') format('woff');
}


html {
  min-height: 100%;
  position: relative;
  color: #fff;
}

html,
body {
  margin: 0;
  padding: 0;
  overflow-x: hidden !important;
  min-height: 100%;
  background: #986cef
}
