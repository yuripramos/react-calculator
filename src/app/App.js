import React, { Fragment } from "react";
import { Provider } from "redux-zero/react";
import { ThemeProvider } from "styled-components";
import "./App.css";
import Routes from "../components/Routes";
import { Router } from "react-router-dom";
// import Modal from "../components/common/Modal";
import { withI18n } from "../utils/i18n";
import history from "../services/history";
import { rem } from "../styles/tools";
// import Footer from "../components/Footer";
import store from "../utils/store";

function App() {
  return (
    <Provider store={store}>
      <Router history={history}>
        <ThemeProvider
          theme={{
            space: [0, rem(5), rem(10), rem(15), rem(20)],
            breakpoints: [rem(767), rem(992), rem(1199)]
          }}
        >
          <Fragment>
            <Routes />
          </Fragment>
        </ThemeProvider>
      </Router>
    </Provider>
  );
}
export default withI18n(App);
