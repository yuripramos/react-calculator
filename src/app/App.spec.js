import React from "react";
import { shallow } from "enzyme";
import createStore from "redux-zero";

import App from "./App";

const store = createStore();

jest.mock("../utils/i18n", () => ({
  translate: id => id,
  withI18n: component => component
}));

describe("App component", () => {
  it("should match snapshot", () => {
    expect(shallow(<App {...store} />)).toMatchSnapshot();
  });
});
