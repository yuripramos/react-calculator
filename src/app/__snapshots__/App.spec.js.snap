// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match snapshot 1`] = `
<Provider
  store={
    Object {
      "getState": [Function],
      "middleware": null,
      "reset": [Function],
      "setState": [Function],
      "subscribe": [Function],
    }
  }
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <ThemeProvider
      theme={
        Object {
          "breakpoints": Array [
            "47.9375rem",
            "62rem",
            "74.9375rem",
          ],
          "space": Array [
            0,
            "0.3125rem",
            "0.625rem",
            "0.9375rem",
            "1.25rem",
          ],
        }
      }
    >
      <React.Fragment>
        <Routes />
        <Footer />
        <ConnectWrapper />
      </React.Fragment>
    </ThemeProvider>
  </Router>
</Provider>
`;
